plugins/inputs/windows_event_log/windows_event_log.go

```diff
@@ -36,6 +36,7 @@ type Plugin struct {
 	FileStateFolder string        `toml:"file_state_folder"`
 	Events          []EventConfig `toml:"event_config"`
 	Destination     string        `toml:"destination"`
+	Log				telegraf.Logger `toml:"-"`
```

Comment:
I think the spacing for these should be consistent, though really only commenting on it because everything above it is consistent.